projects/igniteui-angular-extras/src/lib/context-menu/context-menu.component.scss

```diff
@@ -0,0 +1,129 @@
+@import '../../../../igniteui-angular/src/lib/core/styles/themes/index';
+
+@include igx-fluent-theme($fluent-excel-palette);
```

Comment:
Umm does this generate en entire theme just for this component? 😮

Reply:
Yap, it should be removed

Reply:
done